l5kit/l5kit/environment/callbacks.py

```diff
@@ -145,3 +156,34 @@ def compute_ade_fde(self) -> Tuple[float, float]:
         average_fde = sum(scene_fde_list) / len(scene_fde_list)
 
         return (average_ade, average_fde)
+
+    def compute_driven_miles(self) -> Tuple[float, float]:
```

Comment:
We do have a composite metric for computing driven miles, why not using that one ?

Reply:
Done!